### DISCUSSION TITLE
I can't claim username "KingiLikesMineCraft" even when there is no account with that exact username.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#130045] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I need help, I can't claim username "KingiLikesMineCraft" even when there is no account with that exact username at that same time.
Can anyone help me?
![User with name KingiLikesMineCraft doesn't exist](https://github.com/community/community/assets/167337382/f96e394d-a410-451f-9620-e998e772fa5d)
![And it says that it isn't available](https://github.com/community/community/assets/167337382/1e1c4753-132e-4d22-bcc3-27130e610428)

**Comment by ebndev** [comment]
Hi @KingiLikesMineCraft18291, thanks for being a part of the GitHub Community!

According to [GitHub's Username Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy): ```not all activity on GitHub is publicly visible; accounts with no visible activity may be in active use.``` Perhaps you have created this username in the past or in the chance someone else has, the account might have been banned.

Either way, we are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
